airbyte-integrations/connectors/source-trello/source_trello/schemas/lists.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": [
+    "null",
+    "object"
+  ],
+  "properties": {
+    "id": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "closed": {
+      "type": [
+        "null",
+        "boolean"
+      ]
+    },
+    "pos": {
+      "type": [
+        "null",
+        "number"
+      ]
+    },
+    "softLimit": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "idBoard": {
```

Comment:
is this the same thing as `boardId`?

Reply:
Yes, this is board id. So, by default response, we have an id of board for nested streams.